### ISSUE TITLE
/debug route

### ISSUE THREAD

**Issue by ezrizhu** [stevensblueprint/blueprint_admin#7]
GET /debug shows all the http headers of the request
Currently blocked by #8

**Comment by miguel-merlin** [comment]
To do this task, the react app has to be deployed to the server? @ezrizhu

**Comment by ezrizhu** [comment]
that's correct, i'll also work on nixifying this and deploying it @miguel-merlin

**Comment by miguel-merlin** [comment]
Sounds good

**Comment by ezrizhu** [comment]
Not yet, we'll be tracking the progress on #8, I will have an update by the end of this week.

**Comment by miguel-merlin** [comment]
@ezrizhu Is this unblocked by #11?

**Comment by ezrizhu** [comment]
yes
